tests/unit/visualise/test_generate_map_based_visualisations.py

```diff
@@ -76,3 +86,18 @@ def test_aircraft_track_map_has_children(aircraft_track):
     # Again, this input file is fixed and static so the output should always
     # produce three children.
     assert len(aircraft_track._children) == 3
+
+
+# Tests for get_boundaries:
+def test_boundaries_map_is_Map(boundaries):
+    """Test that boundary data has been successfully converted to a folium
+    Map object."""
+    assert boundaries._name == 'Map'
```

Comment:
A better way to test this is to use `isinstance`:
```suggestion
    assert isinstance(boundaries, folium.Map)
```

Reply:
Done for all 3 functions